client/features/auth/AuthForm.js

```diff
@@ -22,13 +22,16 @@ const AuthForm = ({ name, displayName }) => {
     const formName = evt.target.name;
     const username = evt.target.username.value;
     const password = evt.target.password.value;
-    dispatch(authenticate({ username, password, method: formName }));
+    dispatch(authenticateOld({ username, password, method: formName }));
   };
```

Comment:
what's going on with authenticateOld/New?

Reply:
authenticateOld is for user logging in, authenticateNew is for user signing up so that we can pass in the email when a user signs up